docs/Inside_Arbitrum/ArbGas_and_Fees.md

```diff
@@ -59,9 +59,9 @@ Fees are charged for four resources that a transaction can use:
 * *L2 tx*: a base fee for each L2 transaction, to cover the cost of servicing a transaction
 * *L1 calldata*: a fee per units of L1 calldata directly attributable to the transaction (each non-zero byte of calldata is 16 units, and each zero byte of calldata is 4 units)
 * *computation*: a fee per unit of ArbGas used by the transaction
-* *storage*: a fee per location of EVM storage, based on the net increase in EVM storage due to the transaction
+* *storage*: a fee per location of AVM storage, based on the net increase in EVM storage due to the transaction
```

Comment:
I think we should be consistent with "L2 EVM storage" or "AVM storage"

Reply:
This is meant to refer to storage of EVM contracts that are running on top of Arbitrum/ArbOS.  So EVM storage is closer to correct.